modules/idaithalam/src/main/java/io/virtualan/idaithalam/core/api/ParallelExecutor.java

```diff
@@ -39,18 +40,22 @@ public Integer call() {
             && apiExecutorParam.getInputExcel() != null)
             && !Execution.EXECUTE.name().equalsIgnoreCase(apiExecutorParam.getExecution().name()))
          {
-          ExcelToCollectionGenerator.createCollection(apiExecutorParam);
+       flag=   ExcelToCollectionGenerator.createCollection(apiExecutorParam);
```

Comment:
any reason the flag was added?

Reply:
if ExcelCollectionGenerator is not able to generate the collection, due to some valid reason, we should validate the contract. That's why assigning  the return value to flag and verifying the flag value before procedding